Program.cs

```diff
@@ -21,58 +23,58 @@ static void LengthCheck(string password)
         static void UpperCase(string password)
         {
             int uppercasecounter = 0;
```

Comment:
we don't need to count uppercase characters, just loop over string & find if at least 1 upper case char is present. if yes return on the first occurrence else throw an exception at the end.